spec/02-integration/13-vaults/05-ttl_spec.lua

```diff
@@ -160,8 +164,8 @@ describe("vault ttl and rotation (#" .. strategy .. ") #" .. vault.name, functio
     assert(bp.plugins:insert({
       name = "dummy",
       config = {
-        resp_header_value = fmt("{vault://%s/%s?ttl=%s}",
-                                vault.prefix, secret, 10),
+        resp_header_value = fmt("{vault://%s/%s?ttl=%d&resurrect_ttl=%d}",
```

Comment:
There are now various cases which should be tested in isolation:

* ttl
* ttl _and_ resurrect_ttl
* ttl and neg_ttl and resurrect_ttl
(and all other permutations)

We should keep `ttl` and `ttl + resurrect_ttl` separate IMO.

Reply:
I moved this test to another file.